### PR TITLE
Remove top navbar menu

### PR DESCRIPTION
**Pull request by yubiuser**
**What does this PR aim to accomplish?:**

Removes the top navbar menu. 

Bildschirmfoto zu 2022-08-12 14-50-26

___
The menu does not offer a lot of information and was kind of "hidden".
The links to Github and `pi-hole.net` have been moved to the side bar, Donate was already there (and in the footer). 
Now the session timer is missing, but the default session time is 24 ~~hours~~ minutes (see comments below) to there is no need to be constantly aware of the remaining time. And the link to the latest release has  been removed, but it might be better to link to the discourse announcements instead of the `core` releases only anyway.

Additionally, I moved to logout button to the upper right corner of the top navbar. This is a location where usually logout buttons are placed.

Bildschirmfoto zu 2022-08-12 14-46-19
Bildschirmfoto zu 2022-08-12 14-46-24

Best to be merged after https://github.com/pi-hole/AdminLTE/pull/2290

---
**By submitting this pull request, I confirm the following:** 

1. I have read and understood the [contributors guide](https://docs.pi-hole.net/guides/github/contributing/), as well as this entire template. I understand which branch to base my commits and Pull Requests against. 
2. I have commented my proposed changes within the code and I have tested my changes.
3. I am willing to help maintain this change if there are issues with it later.
4. It is compatible with the [EUPL 1.2 license](https://opensource.org/licenses/EUPL-1.1)
5. I have squashed any insignificant commits. ([`git rebase`](http://gitready.com/advanced/2009/02/10/squashing-commits-with-rebase.html))

---
- [x] I have read the above and my PR is ready for review. _Check this box to confirm_

### CODE REVIEW

**Comment by DL6ER** [comment]
> but the default session time is 24 hours to there is no need to be constantly aware of the remaining time

This is interesting, this has initially been designed for 5 minutes. When staying on the dashboard, the API calls constantly bumped this time so you were never kicked out, but a session time of 24 hours seems pretty useless.

**Comment by rdwebdesign** [comment]
> but the default session time is 24 hours to there is no need to be constantly aware of the remaining time

I think it is configured to 24 minutes.

If you access any page (except the dashboard), you will see the countdown. In the dashboard page the counter is constantly restarted and always shows 23 minutes and 59 seconds.

Anyway, this counter is useless.

**Comment by yubiuser** [comment]
Sorry, 24 minutes is right. It's php default. 

https://www.php.net/manual/en/session.configuration.php#ini.session.gc-maxlifetime

It was always like this: https://github.com/pi-hole/AdminLTE/pull/197/files

**Comment by rdwebdesign** [comment]
My original suggestion was to turn this area into a useful menu, to reduce the main menu, like this:
Menu_suggestion

Note:
I think the counter should be removed too.

**Comment by yubiuser** [comment]
If we keep the menu, we should use an other symbol to indicate that there is a menu. In my opinion the current symbol+logo or the proposed hostname do not indicate strong enough that a menu is hidden there.

**Comment by rdwebdesign** [comment]
I agree.

If we decide to keep the menu, we should use the "hamburger" menu icon 
image


And use different icons for collapse/restore the menu:
image
image

**Comment by dschaper** [comment]
Please remove `Designed for Raspberry Pi`... They've come after us more than once for using their trademarked names.

**Comment by yubiuser** [comment]
I changed to hamburger menu icon to arrows which will change direction on click

File: scripts/pi-hole/js/index.js [hunk 1]
```diff
@@ -1210,3 +1209,8 @@ $(function () {
 window.addEventListener("resize", function () {
   $(".chartjs-tooltip").remove();
 });
+
+// change sidebar icon on collapse/expand
+$(".sidebar-toggle-svg").click(function () {
+  $(this).find("svg").toggleClass("fa-angle-double-left fa-angle-double-right");
+});
```
Comment (rdwebdesign) [inline scripts/pi-hole/js/index.js]: There's no need to use javascript.

You can achieve the same effect using only CSS (in `pi-hole.css`):

```css
.sidebar-collapse .sidebar-toggle-svg {
  transform: scaleX(-1);
}

@media (max-width: 767px) {
  .sidebar-toggle-svg {
    transform: scaleX(-1);
  }
  .sidebar-open .sidebar-toggle-svg {
    transform: scaleX(1);
  }
}
```

Comment (yubiuser) [reply scripts/pi-hole/js/index.js]: Done

**Comment by yubiuser** [comment]
Superseded by https://github.com/pi-hole/AdminLTE/pull/2298

**Pull request closed without merging**